File: rb109_general_ruby_assessment/exercises/101-109_review/easy/tip_calculator.rb
# Source: https://launchschool.com/exercises/90d719d7

# Create a simple tip calculator. The program should prompt for a bill amount
# and a tip rate. The program must compute the tip and then display both the
# tip and the total amount of the bill.

def tip_calculator
  print "What is the bill?\n=> $"
  bill = gets.to_f

  print "What is the tip percentage?\n=> "
  pct = gets.to_f

  tip_amt = ((pct / 100) * bill).round(2)
  total = bill + tip_amt
  puts "The tip is $#{tip_amt}"
  puts "The total is $#{total.round(2)}"
end

tip_calculator

File: exercises/js210_smallproblems/easy_1/tip_calculator.js
let readlineSync = require('readline-sync');

let bill    = Number(readlineSync.question('What is the bill? '));
let tip_pct = Number(readlineSync.question('What is the tip percentage? '));

let tip = (bill * tip_pct / 100);
let total = bill + tip;

console.log(`The tip is $${tip.toFixed(2)}.`);
console.log(`The total is $${total.toFixed(2)}.`);